client/blindai/audio.py

```diff
@@ -1,12 +1,29 @@
 import whisper
 from typing import Optional, Union
-from .utils import fetch_whisper_tiny_20_tokens
 from .client import BlindAiConnection, connect
 from transformers import WhisperProcessor
 import torch
 from ._preprocess_audio import load_audio
+from ._whisper_params import WhisperParams
+from urllib3 import encode_multipart_formdata
+import requests
+import os
+from io import BytesIO
+import numpy as np
 
 DEFAULT_BLINDAI_ADDR = "4.246.205.63"
```

Comment:
This is an Azure server. The Nitro server will have a different IP for obvious reasons,

Reply:
Yeah, I was actually thinking of a Config variable and then we can simply update them when needed.

Reply:
Just have two variables, one for each server.